<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>544</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="result">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QPushButton" name="r_bracket">
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="6">
       <widget class="QSplitter" name="splitter_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="log">
         <property name="text">
          <string>log</string>
         </property>
        </widget>
        <widget class="QPushButton" name="ln">
         <property name="text">
          <string>ln</string>
         </property>
        </widget>
        <widget class="QPushButton" name="sqrt">
         <property name="text">
          <string>sqrt</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="pushButton_1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_0">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="l_bracket">
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="degree">
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSplitter" name="splitter_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>9</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_x">
         <property name="text">
          <string>x</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QSplitter" name="splitter_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="sin">
         <property name="text">
          <string>sin</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cos">
         <property name="text">
          <string>cos</string>
         </property>
        </widget>
        <widget class="QPushButton" name="tan">
         <property name="text">
          <string>tan</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSplitter" name="splitter_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="plus">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
        <widget class="QPushButton" name="minus">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
        <widget class="QPushButton" name="mult">
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QPushButton" name="div">
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="clear_button">
        <property name="text">
         <string>clear</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="mod">
        <property name="text">
         <string>mod</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_dot">
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="credit">
        <property name="text">
         <string>Кредит</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="deposit">
        <property name="text">
         <string>Депозит</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="equal_button">
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QSplitter" name="splitter_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QPushButton" name="asin">
         <property name="text">
          <string>asin</string>
         </property>
        </widget>
        <widget class="QPushButton" name="acos">
         <property name="text">
          <string>acos</string>
         </property>
        </widget>
        <widget class="QPushButton" name="atan">
         <property name="text">
          <string>atan</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="Enter_x">
        <property name="text">
         <string>Enter X</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="x_value"/>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButton_graph">
        <property name="text">
         <string>График</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>320</y>
      <width>150</width>
      <height>116</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>x1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="x1"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>x2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="x2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>y1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="y1"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>y2</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="y2"/>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>291</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
